ctpolicy/ctpolicy.go

```diff
@@ -59,6 +83,40 @@ type result struct {
 	err error
 }
 
+// ExposeLoglistMetrics exposes several prometheus metrics generated by parsing
+// the whole log list. Only needs to be called once because the loglist is not
+// (currently) reloadable. There is no return value other than the scrapable
+// prometheus metrics.
+func (ctp *CTPolicy) ExposeLoglistMetrics() {
```

Comment:
This kind of logic feels kind of funky as an exported method that we call after calling the CTPolicy constructor, `ctpolicy.New()`. What if we moved this logic inside of the constructor instead?

Reply:
It does and I didn't understand that I could have done this all in the constructor. I've since made this much better. Thanks!